wot-wg-2023-draft.html

```diff
@@ -207,8 +207,17 @@ <h2>Motivation and Background</h2>
       <h2>Scope</h2>
       <!-- What exactly is in scope, and out of scope.
           For legal review. Be brief. -->
-      <p>During this charter, the Working Group would work on the following:</p>
 
+      <p>During this charter, the Working Group would work on the following:</p>
+      <p>
+        Interoperability profiles for out-of-the-box interoperability with other ecosystems;
```

Comment:
```suggestion
        Interoperability profiles for out-of-the-box interoperability between greenfield implementations;
```

Profiles do not enable interoperability with other ecosystems, they only provide interoperability within greenfield implementations of a given profile.